src/modules/storage/LedgerStorage.ts

```diff
@@ -299,8 +299,16 @@ export class LedgerStorage extends Storages {
             last_updated_at   INTEGER,
             PRIMARY KEY (address(64))
         );
-        
 
+        CREATE TABLE IF NOT EXISTS account_history(
+            address          TEXT,
+            time_stamp       INTEGER NOT NULL,
+            balance          BIGINT(20) UNSIGNED NOT NULL,
+            granularity      TEXT NOT NULL,
+            
+            PRIMARY KEY (address(64), time_stamp, granularity(64))
+        );
```

Comment:
```suggestion
        CREATE TABLE IF NOT EXISTS account_history(
            address          TEXT,
            block_height     INTEGER NOT NULL,
            time_stamp       INTEGER NOT NULL,
            balance          BIGINT(20) UNSIGNED NOT NULL,
            granularity      TEXT NOT NULL,
            
            PRIMARY KEY (address(64), block_height)
        );
```

Reply:
How about adding an index? (address, time_stamp)

Reply:
mysql create indexes by default on every column in a composite primary key. so there is no need of creating index again.